app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java

```diff
@@ -1990,6 +1990,7 @@ protected Boolean doInBackground(String... arg0) {
                             // if in selection mode, delete selected media
                             if (editMode) {
                                 if (!all_photos && !fav_photos) {
+                                    checkForShare(getAlbum().getSelectedMedia());
```

Comment:
This is added to all if and else if statements and a similar version is present in the else statement. So can you move this out and put it outside the if block?

Reply:
@abishekvashok It needs to be called inside the blocks, as depending on the conditions we need to pass corresponding dataset(ArrayList) as parameter to the function.